apps/route_patterns/lib/route_pattern.ex

```diff
@@ -152,4 +155,15 @@ defmodule RoutePatterns.RoutePattern do
   end
 
   defp stop_ids(_), do: nil
+
+  defp service_id(%{
+         "service" => [
+           %Item{
+             id: service_id
+           }
+         ]
```

Comment:
This arraying of relationships is exactly what is happening in the `line` parsing I am working on.  Should we keep this behavior around?

Reply:
Yeah, I see what you mean.

> Should we keep this behavior around?

Changing this (via removing `list()` in the relevant part of the `%JsonApi.Item{}` struct [here](https://github.com/mbta/dotcom/blob/master/apps/json_api/lib/json_api.ex#L8)) would require a baffling amount of downstream changes in all of our V3 API parsing. I don't see the benefit in changing it.